app-misc/flirc/flirc-3.9.12.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit unpacker
+
+DESCRIPTION="Allows you to pair any remote control with your computer or media center"
+HOMEPAGE="https://flirc.tv/"
+SRC_URI="https://packagecloud.io/Flirc/repo/packages/ubuntu/zesty/flirc_${PV}_amd64.deb/download.deb -> ${P}.deb"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+IUSE="+qt5"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}"
+
+DEPEND=""
+RDEPEND="virtual/libusb
```

Comment:
`virtual/libusb` has two incompatible slots. Given this is a binary package, you need to be specific ;-).